ros/angel_utils/python/angel_utils/conversion.py

```diff
@@ -1,12 +1,14 @@
 """
 Various conversion functions into and out of angel_msg types.
 """
+import array
 from typing import Dict
 from typing import Hashable
 from typing import Iterable
 from typing import List
 from typing import Tuple
 
+import cv2
```

Comment:
This dependency is not represented in this package's `package.xml`. See `ros/angel_system_nodes/package.xml` for an example python OpenCV dep declaration.

Reply:
Ok, I think I added it in the right place.